shark_turbine/kernel/wave/utils.py

```diff
@@ -77,10 +78,12 @@ def DCE(trace: CapturedTrace):
     def is_removable_operator(node: fx.Node) -> bool:
         custom = get_custom(node)
         idxc = IndexingContext.current()
-        is_global_write = (
-            isinstance(custom, Write)
-            and custom.type.address_space.subs(idxc.subs) == GLOBAL_ADDRESS_SPACE
+        is_global_write = isinstance(custom, Write) and (
+            custom.type.address_space.subs(idxc.subs) == GLOBAL_ADDRESS_SPACE
+            or custom.type.address_space.subs(idxc.subs)
+            == tkl.AddressSpace.GLOBAL_MEMORY.value
```

Comment:
Why do we need this comparison test for tkl.AddressSpace.GLOBAL_MEMORY.value ?

Reply:
this is for DCE

Reply:
Currently, the wave e2e tests are using `tkl.AddressSpace.GLOBAL_MEMORY.value` for ADDRESS_SPACE  as convention.